test/onRequest.js

```diff
@@ -66,60 +66,18 @@ describe('Plugin Query Params', function() {
       );
     });
 
-    it('should have lower case query params', function(){
+    it('should have camel case known query params', function(){
       expect(request.query.firstName).to.be.eql(request.query.firstName);
-      expect(request.query.last_Name).to.be.eql(request.query.lastName);
+      expect(request.query.last_Name).to.be.eql(request.query.last_Name);
```

Comment:
this is not camel cased... this is some other horrible casing. I think we either need to specify in the README that the keys passed to the `camelCase` option need to be in the correct case, or camel-case the keys being passed in.

Reply:
I specified in the readMe.